src/controls.h

```diff
@@ -0,0 +1,85 @@
+#pragma once
+#include <utility.h>
+#include <config.h>
+
+
+
+class PID {
+
+    //servo variables
+    //long angle; -> input for servo fn?
+    // bool isAngleUpdate;
+    // long oldPosition=-999;
+
+    //PID data
+    long error;
+    long oldError;
+    long setPoint;
+    long errorInt;
+    float rawValue;
+    long time;
```

Comment:
this looks like the time at which update() was last called. if so, should use a more descriptive name